apps/blockchain/scripts/sync_with_infura.ex

```diff
@@ -95,6 +95,7 @@ defmodule SyncWithInfura do
     Blockchain.Block.add_ommers(block, ommers)
   rescue
     error ->
+      # credo:disable-for-next-line
```

Comment:
Should we use `Logger` instead of `IO` for these kinds of things? I'm guessing that's what credo is complaining about?